packages/orchestration/src/orchestration-api.d.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file Public-facing API of orchestration.
+ * - must not depend on Cosmos types in anyway.
+ * - should remain relatively stable.
+ */
+import type { Amount, Brand } from '@agoric/ertp/exported.js';
+import type { IBCConnectionID } from '@agoric/vats';
+import type { LocalChainAccount } from '@agoric/vats/src/localchain.js';
+import type { ICQConnection } from './exos/icqConnectionKit.js';
+import type {
+  Chain,
+  ChainAmount,
+  ChainInfo,
+  Denom,
+  KnownChains,
+} from './types.js';
+
+/** Details for setup will be determined in the implementation. */
+export interface OrchestrationGovernor {
+  registerChain: (
+    chainName: string,
+    info: ChainInfo,
+    methods?: Record<string, any>,
+  ) => Promise<void>;
+}
+
+// chainName: managed like agoricNames. API consumers can make/provide their own
+export interface Orchestrator {
```

Comment:
I was expecting to see OrchestrationChain and perhaps local chain in here as well. Where are those?

Reply:
That hasn't moved (this is draft) but I'll move it over now